src/configure/configure.ts

```diff
@@ -203,6 +205,8 @@ class Orchestrator {
 
             const repoAnalysisResult = await this.getRepositoryAnalysis();
 
+            TemplateServiceClientFactory.setClientCredentials(this.inputs.azureSession.credentials, this.inputs.githubPATToken);
+            ProvisioningServiceClientFactory.setClientCredentials(this.inputs.azureSession.credentials, this.inputs.githubPATToken);
```

Comment:
move to a separate function named `initializeClients`. This makes it clear to understand that we are initizlizing client at this point in code flow.

Reply:
done